frontend/src/scenes/data-warehouse/new/NewSourceWizard.tsx

```diff
@@ -130,6 +131,8 @@ function FirstStep(): JSX.Element {
     const { featureFlags } = useValues(featureFlagLogic)
 
     const onClick = (sourceConfig: SourceConfig): void => {
+        posthog.capture('new source selected', { sourceType: sourceConfig.name })
+
```

Comment:
Question: Are you missing one of these below for the manual sources since they are handled by a different click listener?